pkgs/development/php-packages/scrypt/default.nix

```diff
@@ -0,0 +1,26 @@
+{ buildPecl, lib, fetchFromGitHub }:
+
+let
+  version = "v2.0.1";
+in buildPecl {
+  inherit version;
+
+  pname = "scrypt";
+
+  src = fetchFromGitHub {
+    owner = "DomBlack";
+    repo = "php-scrypt";
+    rev = version;
+    sha256 = "sha256-JUA8Do43Gmtc0ZWJHBtBzSQ+AhE5nGkiRg8gkVF+Cec=";
```

Comment:
```suggestion
    hash = "sha256-JUA8Do43Gmtc0ZWJHBtBzSQ+AhE5nGkiRg8gkVF+Cec=";
```

Reply:
i could change this, but it doesn't look like the standard in php-packages:
```bash
$ grep -Ri "sha256 =" ./pkgs/development/php-packages | wc -l
51
$ grep -Ri "hash =" ./pkgs/development/php-packages | wc -l
4
```

Reply:
sure, just pushed an update to the commit